src/chapters/css/exercises.rst

```diff
@@ -0,0 +1,13 @@
+Exercises: CSS
+==============
+
+We have built a website for you to test your CSS knowledge on. Check out the `repl.it <https://repl.it/@launchcode/CSSExercises/>`_!
+
+For the exercises, add the following style rules to ``style.css``:
+
+1. Change the background color to yellow.
+2. Change all paragraph text color to red.
+3. Change all ``h1`` to 36 px font size.
+4. Align all text to the center of the page.
+5. Change the font color of elements with the id, ``cool-text``, to blue. 
+
```

Comment:
I noticed that there is a ``script.js`` file in the repl.it and a ``<script>`` element in the html. Were you going to have the students add some JavaScript to the practice page?

Reply:
Nope! Repl.it just wants it there and doesn't like it when I remove it.

Reply:
Ah. I see.

Could we add some code?  If not, maybe add a disclaimer about what the file would be used for if we wanted it.

Reply:
I add a note to it! Thanks!